presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -251,17 +236,13 @@ private void assertUnknownCostForUnknownStats(PlanNode planNode)
                 new FixedLookup(id -> UNKNOWN_COST, id -> UNKNOWN_STATS),
                 session,
                 ImmutableMap.of()))
-                .cpuUnknown()
-                .memoryUnknown()
-                .networkUnknown();
+                .hasUnknownComponents();
         new CostAssertionBuilder(costCalculatorWithEstimatedExchanges.calculateCumulativeCost(
                 planNode,
                 new FixedLookup(id -> UNKNOWN_COST, id -> UNKNOWN_STATS),
                 session,
                 ImmutableMap.of()))
-                .cpuUnknown()
-                .memoryUnknown()
-                .networkUnknown();
+                .hasUnknownComponents();
```

Comment:
`hasUnknownComponents` sound like there are some unknown componets, but here I think you meant that all components are unknown. so I would rename it